src/test/java/com/wikia/webdriver/testcases/templateclassification/TemplateClassificationTest.java

```diff
@@ -1,44 +1,61 @@
 package com.wikia.webdriver.testcases.templateclassification;
 
+import com.wikia.webdriver.common.core.Assertion;
+import com.wikia.webdriver.common.logging.PageObjectLogging;
+import com.wikia.webdriver.elements.oasis.components.templateclassificiation.TemplateClassification;
 import org.testng.annotations.Test;
 import com.wikia.webdriver.common.core.annotations.Execute;
 import com.wikia.webdriver.common.core.annotations.InBrowser;
 import com.wikia.webdriver.common.core.drivers.Browser;
 import com.wikia.webdriver.common.core.helpers.User;
 import com.wikia.webdriver.common.templates.NewTestTemplate;
-import com.wikia.webdriver.elements.oasis.pages.TemplateEditPage;
 import com.wikia.webdriver.elements.oasis.pages.TemplatePage;
 
 @Execute(asUser = User.STAFF, onWikia = "aga")
 @InBrowser(browser = Browser.CHROME)
 @Test(groups = {"templateClassification"})
 public class TemplateClassificationTest extends NewTestTemplate {
 
-  @Test(groups = "templateClassification_openAndClose")
-  public void templateClassification_openAndClose() {
-    new TemplatePage()
-        .open("T")
-        .getTemplateClassification()
-        .open()
-        .close();
-  }
+  private String templateName = "T";
 
-  @Test(groups = "templateClassification_changeTemplateType")
-  public void templateClassification_changeTemplateType() {
-    new TemplatePage()
-        .open("T")
-        .getTemplateClassification()
-        .open()
-        .changeTemplateType()
-        .close();
-  }
+  @Test(groups = "templateClassification_createTemplateAndChangeItsType")
+  public void templateClassification_createTemplateAndChangeItsType() {
+    TemplatePage templatePage = new TemplatePage();
+
+    TemplateClassification templateClassification = templatePage
+            .createTemplate(this.templateName)
+            .open(this.templateName)
+            .getTemplateClassification();
+
+    templateClassification
+            .open()
+            .resetTemplateType()
+            .save();
+
+    Assertion.assertTrue(templateName.equals("Unknown"), "Template type was reset");
+    PageObjectLogging.logInfo("Template type was reset");
+
+    templateClassification
+            .open()
+            .changeTemplateType("Infobox")
+            .save();
+
+    String templateType = templateClassification.getTemplateType();
+
+    Assertion.assertTrue(templateType.equals("Infobox"), "Template type set to Infobox");
+    PageObjectLogging.logInfo("Template type set to: '" + templateType + "'");
+
+    templateClassification
+            .open()
+            .changeTemplateType("Quote")
+            .save();
```

Comment:
What to do you think about wrapping the open -> change -> save sequence in a helper method:
changeTemplateType => selectTemplateType
and leave the entire interaction sequence ( open, select, save ) as the changeTemplateType